### PR TITLE
Feature/recovery neighbor

### PR DESCRIPTION
**Pull request by tjsego**
Contains two add-on modules of modeling tissue recovery. The first model, Simple Recovery, assigns a uniform probability to each dead cell of "resurrecting", where it becomes uninfected again. The second model, Neighbor Recovery, assigns a probability to each dead cell of recovery proportionally to the number of uninfected cells in its neighborhood.

### CODE REVIEW

File: CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Models/RecoveryNeighbor/RecoverySteppables.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        self.rec_steppable_key = rec_steppable_key
+
+    def cell_recovers(self, _cell) -> bool:
+        """
+        Recovery criterion
+        :param _cell: dead cell to test for recovery
+        :return: True if cell recovers
+        """
+        ca = sum([a for n, a in self.get_cell_neighbor_data_list(_cell) if n is not None and n.type == self.UNINFECTED])
+        return random.random() < ca * recovery_rate * s_to_mcs
```
Comment (JulianoGianlupi) [inline CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Models/RecoveryNeighbor/RecoverySteppables.py:56]: so prob_recovery isn't \proto number neighbors but common area in this case, correct?

**Review by JulianoGianlupi — APPROVED** [review]

File: CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Models/RecoveryNeighbor/RecoverySteppables.py (hunk 1 above)
Comment (joaponte) [reply CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Models/RecoveryNeighbor/RecoverySteppables.py:56]: Good point!

**Review by joaponte — APPROVED** [review]
This is nicely organized

File: CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Models/RecoveryNeighbor/RecoverySteppables.py (hunk 1 above)
Comment (tjsego) [reply CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Models/RecoveryNeighbor/RecoverySteppables.py:56]: Correct. This has been noted in documentation of the inputs

**Pull request merged**